src/main/java/io/milvus/param/DeleteParam.java

```diff
@@ -0,0 +1,60 @@
+package io.milvus.param;
+
+import javax.annotation.Nonnull;
+
+public class DeleteParam {
+    private final String dbName;
```

Comment:
We don't have primary field for delete?

Reply:
proto doesn't have primary filed,but proto has value expr. add value expr in new commit